coreneuron/sim/scopmath/crout_thread.cpp

```diff
@@ -50,11 +50,12 @@ static char RCSid[] = "crout.c,v 1.2 1999/01/04 12:46:43 hines Exp";
 /*--------------------------------------------------------------*/
 
 #include <math.h>
-#include "coreneuron/scopmath_core/errcodes.h"
+#include "coreneuron/sim/scopmath/errcodes.h"
 namespace coreneuron {
 #define ix(arg) ((arg)*_STRIDE)
 
 /* having a differnt permutation per instance may not be a good idea */
+#pragma acc routine seq
```

Comment:
curious how/why this pragma is introduced here

Reply:
This I introduced with 1fef969 to fix the failing build that you mentioned above. I saw that it did not fix it, but I believe this directive is still needed:
https://www.pgroup.com/lit/articles/insider/v6n1a1.htm
or am I missing something?

The build failure still needs looking into, but honestly I'm not quite sure what I changed that broke this.

Reply:
discussed and fixed.